.github/workflows/sample-apps-with-v2-beta.yml

```diff
@@ -0,0 +1,24 @@
+name: Building sample apps with v2-beta action
```

Comment:
I wanted a way for us to readily look to see the status of our existing github actions against few samples. Initially I wanted to create few sample apps in the actions repo itself but that might increase the pull time of the actions for actual users, so instead creating it here.

Reply:
We should probably also have an end-to-end sample of using Oryx to build and deploy to a sample Azure Web App site. This way end users would have a quick sample to look at it and get a feel of an E2E with Oryx.

Reply:
I don't think we should necessarily bind the name of this file to the version of the action we're using -- if this workflow is used for testing our action against different scenarios, I think the workflow would look something like the following:

- Release a new version of our action
- Open a pull request against `master` from a new `actions/**` branch that updates this workflow to reflect the new version
- Verify that the kicked-off actions build succeeds and merge the pull request

Do you think that there are scenarios where we would want to support multiple versions of the action at once? If so, we could update this workflow to kick off multiple runs per build that would each test a different version of the action.

Reply:
That makes sense -- I think we can get away with having a single workflow file that kicks off a single job with multiple runs, each testing a different version of the action that we're interested in. Initially, the file might be large and repetitive since we would be testing the same apps across different action versions, but the GitHub Actions team said they're looking into some form of templating (similar to what we do now for ADO). Having a single workflow will make it easier to track all of the runs for a given branch push (or any additional triggers we may add).